pureskillgg_dsdk/tome/header_tome.py

```diff
@@ -36,20 +37,21 @@ def create_header_tome_from_fs(
     )
 
     writer = TomeWriterFs(root_path=tome_collection_root_path, tome_name=name, log=log)
-    manifest = TomeManifest(
+    manifest_tome = TomeManifest(
         tome_name=name, path=writer.path, ds_type=ds_type, is_header=True
     )
-    scribe = TomeScribe(manifest=manifest, writer=writer, log=log)
+    scribe = TomeScribe(manifest=manifest_tome, writer=writer, log=log)
 
     scribe.start()
 
-    for ds_path in get_ds_paths_from_glob(ds_collection_root_path, path_depth, ds_type):
-        ds_loader = fetch_ds_header_from_fs(ds_path, ds_type, log)
+    for manifest_key in get_manifest_keys_from_glob(ds_collection_root_path, ds_type):
+        ds_loader = fetch_ds_loader_from_fs(ds_collection_root_path, manifest_key, log)
         job_id = ds_loader.manifest["jobId"]
```

Comment:
Maybe we should retire `jobId` in the dsdk in favor of just `match_id`. So this would just be
```
         match_id = ds_loader.manifest["id"]
```

Reply:
👍